src/XMakeTasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1602,7 +1602,7 @@ private void LogAssembliesConsideredAndRejected(Reference reference, MessageImpo
                                 break;
                             }
                         case NoMatchReason.FusionNamesDidNotMatch:
-                            Log.LogMessageFromResources(importance, "ResolveAssemblyReference.EightSpaceIndent", Log.FormatResourceString("ResolveAssemblyReference.ConsideredAndRejectedBecauseFusionNamesDidntMatch", location.FileNameAttempted, location.AssemblyName.FullName));
+                            Log.LogMessageFromResources(importance, "ResolveAssemblyReference.EightSpaceIndent", Log.FormatResourceString("ResolveAssemblyReference.ConsideredAndRejectedBecauseFusionNamesDidntMatch", location.FileNameAttempted, location.AssemblyName.FullName, location.AssemblyName.Name));
```

Comment:
That's a good point. @nikson, can you include sample output for a mismatch here? That'll help figure out if it's emitting the right thing.